extensions_admin/pulp_rpm/extensions/admin/status.py

```diff
@@ -330,3 +333,18 @@ def update_func(new_state):
         render_general_spinner_step(self.prompt, self.comps_spinner, current_state,
                                     self.comps_last_state,
                                     _('Importing package groups/categories...'), update_func)
+
+    def render_purge_duplicates_step(self, progress_report):
+        # Example Data:
+        # "purge_duplicates": {
+        #   "state": "FINISHED"
+        # }
```

Comment:
This is [a foolish consistency](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds), and looks like the docblocks of all similar methods in this module as a result.